src/ydata_synthetic/synthesizers/loss.py

```diff
@@ -5,9 +5,9 @@
 from enum import Enum
 
 class Mode(Enum):
-    WGANGP = 'wgangp'
+    WGAN_GP = 'wgangp'
     DRAGAN = 'dragan'
-    CRAMER = 'cramer'
+    CRAMERGAN = 'cramer'
```

Comment:
Why the name change?

Reply:
The idea is to make it consistent with the `__MODEL__` attribute.
It was defined at class level but in some cases we were not leveraging it.
If we make the enum consistent with this attribute we can generalize calls like `an_enum_class[self.__MODEL__]` instead of _ad hoc_ specifications like `an_enum_class.A_MODEL_CLASS` . (I am thinking on leveraging this attribute in enums for the supported models of the DataProcessors).
Since the enum was not consistent with this attribute the name change could have been done on the attribute itself or the enum (I chose the latter).
Let me know if it makes sense to you!